docs/source/notebooks/release_announcement.ipynb

```diff
@@ -0,0 +1,2774 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    model.plot_components_contributions();
</pre><p>We would also add a final sentence along the lines: "We expect to add different media transformation parametrizations and more diagnostics plots in the next releases" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Added footnote</p>